nix/README.md

```diff
@@ -0,0 +1,50 @@
+# Nix build support facilities
+
+## Development Environment
+
+The dependencies for a dynamic built can be fetched with
+```sh
+nix develop
+```
+
+The `direnv` tool has good support for nix and can be used with an `.envrc` with
+the content:
```

Comment:
Why would I do that? It would be nice to shed a bit of insight for the use case here, e.g., "to do X, use Y."

Reply:
I'm not saying we should write documentation for other tools.

My point is: simply tell the user why I should care. I've never heard of `direnv`. Why do I care and need it here?